pysatNASA/tests/test_instruments.py

```diff
@@ -34,13 +42,18 @@
 instruments = clslib.InstLibTests.initialize_test_package(
     clslib.InstLibTests, inst_loc=pysatNASA.instruments)
 
+# Additional tests required for pandas instruments if pysatCDF installed
 # Create a new list of instruments with the option of forcing cdflib
 instruments['cdf'] = []
+
+# Create list of pandas instruments where this is not needed
+skip_cdf_list = ['de2_vefimagb']
```

Comment:
For future users and maintainability, is there documentation somewhere about this?

Reply:
I'm pretty sure there's an issue about this at pysatCDF. :)

Technically we don't run any tests with pysatCDF anymore. See https://github.com/pysat/pysatNASA/issues/167

Reply:
Lame but I get it.